gf-webapp-admin/src/component/sideMenu/addRoomModal/AddRoomModal.tsx

```diff
@@ -24,26 +26,97 @@ interface AddRoomModalProps {
 export const AddRoomModal = (props: AddRoomModalProps) => {
 
   const [room, setRoom] = useState<NewRoom>(defaultRoom);
+  const [roomServices, setRoomServices] = useState(room.services);
+  room.services = roomServices;
```

Comment:
is it ok to put roomServices in some object? React doesn't see that!

Reply:
done